test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageReaderTests.cs

```diff
@@ -497,5 +497,20 @@ public void PackageReader_SupportedFrameworks()
                 }
             }
         }
+
+        [Fact]
+        public void PackageReader_SupportedFrameworksForInvalidPortableFrameworkThrows()
```

Comment:
In theory this test is not required, but since https://github.com/NuGet/Home/issues/1869 mentions a broken package this test was added to test front to back.